app/models/camaleon_cms/post_default.rb

```diff
@@ -3,6 +3,15 @@ class PostDefault < CamaleonRecord
     include CamaleonCms::Metas
     include CamaleonCms::CustomFieldsRead
 
+    def self.inherited(subclass)
+      super
+
+      # instance_eval so the code is eval'd in the context of subclass, not subclass's class, which would be Class 🤪
```

Comment:
Please remove my incorrect comment! 😅 Sorry about that.

Reply:
:) NP

Fixed in https://github.com/owen2345/camaleon-cms/pull/1046/commits/2bf6e26fa8a153dd94c2661d03cf481fba0ed9ed